### PR TITLE
feat: add solution to problem [implement-omit-t-k]

### PR DESCRIPTION
**Pull request by jiechen257**

### CODE REVIEW

File: typescript/implement-omit-t-k_en.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
+// @jichen257
 
-There is no solution yet.
-
-Would you like to [contribute to the solution](https://github.com/BFEdev/BFE.dev-solutions/blob/main/typescript/implement-omit-t-k_en.md)? [Contribute guideline](https://github.com/BFEdev/BFE.dev-solutions#how-to-contribute)
+Pay attention to `K extends keyof any`, K constraint is the key type of any Object
+```ts
+type MyOmit<T, K extends keyof any> = {
+  [P in keyof T as P extends K ? never : P]: T[P]
```
Comment (BFEdev) [inline typescript/implement-omit-t-k_en.md:7]: Thanks for contributing, could you add a bit more explanation on here ?
it's also worthy it to mention another approach `[P in Exclude<keyof T, K>]: T[P]`?

File: typescript/implement-omit-t-k_en.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
+// @jichen257
 
-There is no solution yet.
-
-Would you like to [contribute to the solution](https://github.com/BFEdev/BFE.dev-solutions/blob/main/typescript/implement-omit-t-k_en.md)? [Contribute guideline](https://github.com/BFEdev/BFE.dev-solutions#how-to-contribute)
+Pay attention to `K extends keyof any`, K constraint is the key type of any Object
```
Comment (BFEdev) [inline typescript/implement-omit-t-k_en.md]: > `key type of any Object` 

Please make it clear here that it is `string | number | symbol`

**Review by BFEdev — CHANGES_REQUESTED** [review]
Thanks for contributing,

**Comment by jiechen257** [comment]
> Thanks for contributing,

Thank you for the suggestion. I have corrected and submitted it, please review again

**Pull request merged**